c7n/filters/core.py

```diff
@@ -330,6 +331,19 @@ def process_value_type(self, sentinel, value):
             # comparisons is intuitively wrong.
             return value, sentinel
 
+        elif self.vtype == 'cidr_size':
+            v = []
+            for r in value:
+                try:
+                    cidr_size = ipaddress.ip_network(unicode(r.get('CidrIp'))).prefixlen
```

Comment:
i think alot of this handling can be abstracted out to utils with common usage from vpc and the value_types.

common exception handling defers to the vpc action/filters needs to report errors and continue optionally.